pkg/provisioner/ironic/ironic.go

```diff
@@ -1347,6 +1383,10 @@ func (p *ironicProvisioner) Provision(hostConf provisioner.HostConfigData) (resu
 		return transientError(provisioner.NeedsRegistration)
 	}
 
+	if err = p.updateAutomatedClean(ironicNode); err != nil {
+		return transientError(errors.Wrap(err, "could not update ironic node automated_clean field"))
```

Comment:
nit: I'd prefer this to be phrased in terms of metal3 entities, not ironic fields

Reply:
Sure, will update it.